rskj-core/src/main/java/org/ethereum/core/TransactionExecutor.java

```diff
@@ -322,6 +323,7 @@ private void call() {
         this.subtraces = new ArrayList<>();
 
         if (precompiledContract != null) {
+            this.precompiledContractHasBeenCalledFlag = true;
```

Comment:
Isn't this detected in `play`?

Reply:
If the transaction calls a Precompiled contract, `play` is not executed.